app/helpers/patient_query_helper.rb

```diff
@@ -475,6 +478,34 @@ def advanced_filter_quarantine_option(patients, filter, tz_offset, option_type)
     filter[:value].present? ? query : patients.where.not(id: query.pluck(:id))
   end
 
+  def advanced_filter_lab_result(patients, filter)
+    labs = patients.joins(:laboratories)
+    filter[:value].each do |field|
+      case field[:name]
+      when 'lab-type'
+        labs = labs.where('laboratories.lab_type = ?', field[:value])
+      when 'result'
+        labs = labs.where('laboratories.result = ?', field[:value])
+      when 'specimen-collection'
+        case field[:value][:when]
```

Comment:
I forget if I asked this before, but does the PO team now want a `between` option for dates?

Reply:
That is not supposed to be a part of this iteration

Reply:
sorry, I don't mean to bring up something that's been resolved already, but did the PO team discuss this in a thread? I was just looking through the mockups and did noticed that the `between` option was requested as well as relative date filters. I'm sure you're right about those being in a future iteration, but was just wondering if there's a clear DoD for which features belong in this first iteration

![Screen Shot 2021-04-21 at 12 23 44 AM](https://user-images.githubusercontent.com/9956561/115496548-e3949a80-a237-11eb-9d85-283b7230aabd.png)

Reply:
Totally forgot to add this here, but this was discussed with the PO team and will not be included at this present time